build.gradle

```diff
@@ -347,6 +368,12 @@ shadowJar {
     configurations = [project.configurations.runtime]
     classifier = 'local'
     mergeServiceFiles('reference.conf')
+    doLast {
+        // Create a symlink to the newly created jar.  The name will be gatk.jar and
+        //  it will be at the same level as the newly created jar.  (overwriting symlink, if it exists)
+        // Please note that this will cause failures in Windows, which does not support symlinks.
+        createAllSymlinks(destinationDir.toString(), archivePath, "")
+    }
 }
 
 task localJar{ dependsOn shadowJar }
```

Comment:
It seems to me that perhaps is more appropriate that ```shadowJar``` invokes ```localJar``` rather than the opposite in order to make is more coherent with ```sparkJar``` as ```local``` is better contrasting term (classifier?) to ```spark```.